Doc/library/os.rst

```diff
@@ -3945,8 +3945,14 @@ written in Python, such as a mail server's external command delivery program.
    documentation for more powerful ways to manage and communicate with
    subprocesses.
 
-   .. versionchanged:: 3.11
-      Added the *encoding* parameter.
+   .. note::
+      On Unix, the *cmd* argument is encoded with
+      :term:`file system encoding <filesystem encoding and error handler>`.
+      The returned file object connected to the pipe uses
+      :func:`locale.getpreferredencoding(False) <locale.getpreferredencoding>`_.
+      The :ref:`Python UTF-8 Mode <utf8-mode>` affects both encodings.
+
+      Use :class:`subprocess.Popen` to control encoding and many other options.
```

Comment:
The last sentence "... use subprocess to control encoding ..." is fine. But I'm not sure that it's worth it to maintain a documentation of the encoding used by subprocess here. I'm not against it ;-)